### DISCUSSION TITLE
Philips LCG007 unsupported bulb

### DISCUSSION THREAD

**Discussion by CtznSniiips** [Koenkk/zigbee2mqtt#24269] [Devices]
I got some Philips bulbs model LCG007 which were added to zigbee2mqtt and all exposes work correctly, but it's listed as "unsupported". This isn't really a problem since everything works correctly, but the bulbs are all showing a firmware update available. 

Would it be wise to skip the firmware update given these are listed as unsupported? How could it retrieve firmware for an unsupported device?

![Screenshot 2024-10-08 151445](https://github.com/user-attachments/assets/a757a0e6-bb16-4363-b220-3957047a3f6d)

**Comment by Koenkk** [comment]
Could you try with the [generated external converter](https://www.zigbee2mqtt.io/advanced/support-new-devices/01_support_new_devices.html#_2-creating-the-external-definition) instead?

**Reply by CtznSniiips** [reply]
Should it say supported after I press "Generate external definition"? Nothing seems to have changed.

**Reply by Koenkk** [reply]
I can add it, could you provide me a link to the product page?

**Reply by CtznSniiips** [reply]
I bought both the 1-pack and the 2-pack on Amazon, they were all identified the same. https://www.amazon.com/dp/B0D4XNZVSP?ref=ppx_yo2ov_dt_b_product_details&th=1

This is the product on Philips website: https://www.philips-hue.com/en-us/p/hue-white-and-color-ambiance-gu10---smart-spotlight/046677584658

**Reply by Koenkk** [reply]
Added!

Changes will be available in the [dev branch](https://www.zigbee2mqtt.io/advanced/more/switch-to-dev-branch.html) in a few hours from now.
